source/opt/insert_extract_elim.h

```diff
@@ -40,24 +40,29 @@ class InsertExtractElimPass : public Pass {
   Status Process(ir::IRContext*) override;
 
  private:
-  // Return true if indices of extract |extInst| starting at |extOffset|
+  // Return true if indices in |extIndices| starting at |extOffset|
   // match indices of insert |insInst|.
```

Comment:
It is also not clear how `extIndices` is to be interpreted.  I was able for figure it out because I knew it was replacing the indices from in an extract instruction.  Might be worth making the comments a little clearer.

Reply:
K